frontend/packages/console-shared/src/components/TechPreviewBadge.tsx

```diff
@@ -2,7 +2,9 @@ import * as React from 'react';
 import { Label } from '@patternfly/react-core';
 
 const TechPreviewBadge: React.FC = () => (
-  <Label style={{ backgroundColor: 'var(--pf-global--warning-color--100)' }}>Tech Preview</Label>
+  <Label style={{ maxWidth: '120px', backgroundColor: 'var(--pf-global--warning-color--100)' }}>
```

Comment:
This shouldn't have a `max-width`.
I see why you've done this.
Move it beside the title so that it looks like the mocks:
![image](https://user-images.githubusercontent.com/14068621/61922919-19f09900-af30-11e9-905b-cfc12b9b8179.png)

Reply:
This component is used for the same badge in the CLI help screen. So please double check that as you make changes.